torch/csrc/jit/passes/quantization.cpp

```diff
@@ -421,6 +424,17 @@ void InsertObserversHelper::insertObservers(
 
   script::Method method = module.get_method(method_name);
   auto graph = method.graph();
+
+  if (graph_observer_map_.count(graph.get())) {
+    // instance clone of observer module and setAttr
+    for (const auto& observer_attrs : graph_observer_map_.at(graph.get())) {
+      const auto& name = std::get<0>(observer_attrs);
+      const auto& observer = std::get<1>(observer_attrs);
+      module._ivalue()->setAttr(name, observer.clone_instance()._ivalue());
+    }
+    return;
+  }
```

Comment:
This condition should probably be moved to the beginning of the function. There is no need to repeat this work for every invoked method.

Reply:
what do you mean? 
We didn't repeat this part for every invoked method, this is outside of the loop of calling insertObservers for invoked methods.
this is intentionally put after we call insertObservers for all invoked methods.